grafana-plugin/src/pages/integrations/Integrations.tsx

```diff
@@ -122,6 +128,7 @@ class Integrations extends React.Component<IntegrationsProps, IntegrationsState>
   }
 
   componentWillUnmount() {
+    this.isMounted = false;
     clearInterval(this.alertReceiveChannelTimerId);
```

Comment:
hmm, timer is cleared here, didn't get why should we check `this.isMounted`...

Reply:
That has nothing to do with timer, it's about when you move from integrations to a separate page. Once it loads the escalation chains it calls that method but you're on a different page already.